README.md

```diff
@@ -11,9 +11,9 @@ The definitive JNA reference (including an overview and usage details) is in the
 
 JNA provides Java programs easy access to native shared libraries without writing anything but Java code - no JNI or native code is required. This functionality is comparable to Windows' Platform/Invoke and Python's ctypes.
 
-JNA allows you to call directly into native functions using natural Java method invocation. The Java call looks just like the call does in native code. Most calls require no special handling or configuration; no boilerplate or generated code is required.
+JNA allows you to call directly into native functions using the natural Java method invocation. The Java call looks just like the call does in native code. Most calls require no special handling or configuration; no boilerplate or generated code is required.
```

Comment:
I don’t think the additional article here helps and in fact makes this less readable.